### DISCUSSION TITLE
Temporary adventurer notes

### DISCUSSION THREAD

**Discussion by sylsau** [organicmaps/organicmaps#5221] [Ideas]
In a self-adventurer fashion, I was looking for a place on the internet where people could share notes pinned to the map on the topics of their choice.

I haven't found it, so I thought "it could be a feature of an app I already know!" OrganicMaps was the natural candidate.

Examples of possible notes:
- "The otters there are unruly!"
- "Next to this big tree you'll find a cool-looking abandoned wood cabin"
- "I must have left a picnic bag there... Has anyone found anything?"

It'd just be another category showing these special notes à la OSM. It could be titled "Adventurer Notes" or something less fancy.

Now, to avoid flooding and also add a layer of mystery, the notes would have to disappear after some time. I'm thinking one month to one day could be a good amount of time to let some people explore the possibilities without the map becoming unreadable or the feature turning into pseudo-Facebook walls.
There's also the option of restricting the number of contribution to 2 per week maximum, as to, again, avoid flooding.

Obviously these are not yet very elaborate guesses but I imagine it could recreate a sense of wonder on a local scale and at the same time energize the community of users of the app.

I realize it's a simple thing to ask for features to developers who might otherwise be very busy, but I must say I love your software and have been thinking about this for some time now so I hope you'll appreciate the idea.

Thank you for your work!

**Comment by biodranik** [comment]
That's a very cool idea! Although a robust spam-protection mechanism should be designed for it :)
